### PR TITLE
Update 13 + 5.4

### PR DESCRIPTION
**Pull request by Rainb0wKey**
yes yes yes yes update!

### CODE REVIEW

File: src/bdscript/banID.md [hunk 1]
```diff
@@ -1,38 +1,16 @@
 # $banID
-Bans a user using their ID.
-
-## Usages
-There are three usages of the `$banID` function.
-
-### Usage #1
+Bans a user using their ID without reason.
```
Comment (MineBartekSA) [inline src/bdscript/banID.md:2]:
```suggestion
Bans a user using their ID without reason.
The user ID will be taken from the last part of the author's message.

```

File: src/bdscript/banIDComplex.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# $banID[]
+Bans a user using their ID.
+## Syntax
+```
+$banID[reason;(user ID)]
+```
+## Parameters
+- `reason`: The reason for this action, which will be saved in the audit-log.
+> Use `$getBanReason[]` to get the ban reason.
+- `user ID`: Gets the user to ban. _(Optional)_
```
Comment (MineBartekSA) [inline src/bdscript/banIDComplex.md]:
```suggestion
- `user ID`: The user to ban. If empty, the ID will be taken from the last part of the author's message. _(Optional)_
```

File: src/bdscript/changeUsernameWithID.md [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
-$changeUsernameWithID[userID;newNickname]
+$changeUsernameWithID[user ID;new nickname]
 ```
-
-### Breakdown
-- `userID` - The ID of the user whose nickname will be changed.
-- `newNickname` - The text to change the user's nickname to. It cannot exceed more than 32 characters.
-
+## Parameters
+- `user ID`: The ID of the user whose nickname will be changed.
```
Comment (MineBartekSA) [inline src/bdscript/changeUsernameWithID.md]:
```suggestion
- `user ID`: The user whose nickname will be changed.
```

File: src/bdscript/channelExists.md [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
+## Syntax
 ```
-$channelExists[channelID]
+$channelExists[channe lID]
 ```
-
-### Breakdown
-- `channelID` - The channel which the bot will check for.
+## Parameters
+- `channel ID`: The channel which the bot will check for.
```
Comment (MineBartekSA) [inline src/bdscript/channelExists.md]:
```suggestion
- `channel ID`: The channel for which the bot will check for.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Comment by Rainb0wKey** [comment]
Ready^

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**